### PR TITLE
Update buildGlibcWASM.md

### PR DESCRIPTION
**Pull request by robinyuan1002**

### CODE REVIEW

File: docs/buildGlibcWASM.md [hunk 1]
```diff
@@ -122,17 +127,62 @@ In the build directory, usually we use `make --keep-going -j$(nproc)`. The first
 This procedure is specified in the `gen_sysroot.sh` script in our glibc repo. It's main job is to generate a WASM sysroot structre like
 
 ```
-sysroot/
```
Comment (yzhang71) [inline docs/buildGlibcWASM.md:125]: This is the structure of WASM sysroot and should keep this part

File: docs/buildGlibcWASM.md [hunk 2]
```diff
@@ -122,17 +127,62 @@ In the build directory, usually we use `make --keep-going -j$(nproc)`. The first
 This procedure is specified in the `gen_sysroot.sh` script in our glibc repo. It's main job is to generate a WASM sysroot structre like
```
Comment (yzhang71) [inline docs/buildGlibcWASM.md:127]: "structre" is a typo

**Review by yzhang71 — COMMENTED** [review]
Please address the comments I made

File: docs/buildGlibcWASM.md (hunk 1 above)
Comment (robinyuan1002) [reply docs/buildGlibcWASM.md:125]: solved

**Pull request merged**